dwave/embedding/chain_strengths.py

```diff
@@ -37,6 +37,22 @@ def uniform_torque_compensation(bqm, embedding=None, prefactor=1.414):
     Returns:
         float: The chain strength, or 1 if chain strength is not applicable
                to the problem. 
+
+    Examples:
```

Comment:
Wonder if it makes sense to move these examples into the module-level docstring. Since it will likely be repeated for each function. We should also add these functions to the docs.